R/check.R

```diff
@@ -56,11 +60,12 @@ check_reverse_dependencies <- function(
     output = tempfile(paste(utils::packageName(), Sys.Date(), sep = "-")),
     lib.loc = .libPaths(),
     repos = getOption("repos"),
+    reverse_repos = repos,
```

Comment:
I added this to the high level interface to make sure users using the API can easily utilize binaries on Linux

Reply:
fine for now, though I feel like it doesn't really capture what we want the argument to do.

It's not obvious just from the argument that this implies one can use binaries and one can use source. 

We can think about what a more intuitive interface might look like. I'm imagining something like:

```r
  repos = list(source = "...", binary = "..."),
  type = c("binary", "source")  # prefer binary, use source only as fall back or when needed
```